spacemesh/v2alpha1/layer.proto

```diff
@@ -0,0 +1,50 @@
+syntax = "proto3";
+
+import "spacemesh/v2alpha1/block.proto";
+
+package spacemesh.v2alpha1;
+
+message Layer {
+  oneof versioned {
+    LayerV1 v1 = 1;
+  }
+}
+
+message LayerV1 {
+  uint32 number = 1; // layer number - not hash - layer content may change
+  enum LayerStatus {
+    LAYER_STATUS_UNSPECIFIED = 0;
+    LAYER_STATUS_APPLIED = 1; // applied by hare
+    LAYER_STATUS_VERIFIED = 2; // verified by tortoise
+  }
+  LayerStatus status = 2;
+  string consensus_hash = 3;
+  bytes state_hash = 4; // fingerprint of the computed state at the layer
+  bytes cumulative_state_hash = 5; // cumulative fingerprint that uniquely identifies state since genesis
+  Block blocks = 6; // layer's blocks
```

Comment:
This is confusing. The plural suggests there's more than one, but `Block` is only a single block. See also #287.

Reply:
Changed